### ISSUE TITLE
Recurrent error during update process on 0.9.7 emails

### ISSUE THREAD

**Issue by ZeHiro** [cozy/cozy-emails#588]
```
/usr/local/cozy/apps/emails/build/server/models/message.js:392

    if ((ref = box.id, indexOf.call(hasTwin, ref) >= 0) && indexOf.call(twinMa
                                                                                                      ^

TypeError: Array.prototype.indexOf called on null or undefined
    at indexOf (native)
    at Message.module.exports.Message.markTwin (/usr/local/cozy/apps/emails/build/server/models/message.js:392:68)
    at /usr/local/cozy/apps/emails/build/server/models/message.js:359:25
    at /usr/local/cozy/apps/emails/build/server/models/message.js:174:14
    at /usr/local/cozy/apps/emails/node_modules/cozydb/lib/cozymodel.js:254:18
    at parseBody (/usr/local/cozy/apps/emails/node_modules/cozydb/node_modules/request-json-light/main.js:75:10)
    at IncomingMessage.<anonymous> (/usr/local/cozy/apps/emails/node_modules/cozydb/node_modules/request-json-light/main.js:109:14)
    at IncomingMessage.EventEmitter.emit (events.js:117:20)
    at _stream_readable.js:920:16
    at process._tickCallback (node.js:415:13)
```
Emails is then restarted

**Comment by aenario** [comment]
This should have been fixed in just released 0.9.8 version

**Comment by ZeHiro** [comment]
Effectivement. Merci.
